RELEASING.md

```diff
@@ -6,29 +6,42 @@ The following are step-by-step instructions for releasing the Promscale extensio
 
 Create a new issue titled "Release <version to be released here>". Copy everything below this line into that issue, and use it to to keep track of the release tasks.
 
---- 
-
 ## Pre-release
+- [ ] Create a git branch named `pre-release-x.x.x`
 - [ ] Ensure `upgradeable_from` in `templates/promscale.control` contains the previously released version.
 - [ ] Update `CHANGELOG.md` with release version and date.
-- [ ] Update the extension version in `Cargo.toml`, don't forget to fix the `.so` references in `incremental/001-extension.sql`, `idempotent/014-extension-type-functions.sql`, and `promscale--0.0.0.sql`.
-- [ ] Freeze the versions of SQL scripts in `incremental_freeze_test`
-- [ ] Update the extension tag in the _Compile From Source_ instructions.
-- [ ] Merge release commit.
+- [ ] Update the package version in `Cargo.toml` to the version being released
+- [ ] Update the version in the references to the .so in `migration/incremental/001-extension.sql`
+- [ ] Update the MD5 hash of the `migration/incremental/001-extension.sql` file in `e2e/tests/incremental-freeze-test.rs`
+- [ ] Update the version in the references to the .so in `migration/idempotent/014-extension-type-functions.sql`
+- [ ] Update the version in the references to the .so in `sql/promscale--0.0.0.sql`
+- [ ] Freeze any new incremental sql scripts released in this version in `incremental_freeze_test`
+- [ ] Update the extension tag in the _Compile From Source_ instructions in `INSTALL.md` (there are more than one!)
+- [ ] Create a PR from the `pre-release-x.x.x` branch. Get it approved. Merge the PR to master.
 
 ## Release
-- [ ] Tag and push merged release commit.
-  - CI will trigger and create a draft release with assets attached.
+- [ ] [Create a draft release in GitHub](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) off the master branch with a new Tag pointing to the version
```

Comment:
I'm not sure that this this is necessary, and I'm not sure how the draft release that one creates here interacts with the draft release which is created by CI (do you?).

At this point the "only thing" that needs to be done if for the tag to be created. And actually the tag is the important thing, because it kicks off the release build in CI. It seems as though this describes using github's "create release" mechanism to create a tag, but one could also do:

```
git tag <new version>
git push origin <new version>
```

Reply:
To be honest, I was stumbling through this part because the prior instructions were very vague, so I'm not entirely sure how it was supposed to work. I did it by creating a tag in the github ui while creating a draft release. If the cmd line approach does the same thing, then I'd rather document the command line approach.